api/v1alpha1/flowcollector_types.go

```diff
@@ -39,40 +39,42 @@ const (
 type FlowCollectorSpec struct {
 	// Important: Run "make generate" to regenerate code after modifying this file
 
-	// Namespace where NetObserv pods are deployed.
+	// namespace where NetObserv pods are deployed.
 	// If empty, the namespace of the operator is going to be used.
 	// +optional
 	Namespace string `json:"namespace,omitempty"`
 
+	// agent for flows' extraction.
```

Comment:
nit / typo: I don't think the quote is needed here? Should be just "gent for flows' extraction" ?
(If a native english speaker could confirm, cc @nalhadef / @stleerh ... )

Reply:
Not 100% sure but I think you are right. Fixing it and we can revert it back if Neal/Steven or anyone else says it's required.

Reply:
@mariomac @jotak I agree. The quotation marks should be removed.